esy/src/lib/child_process.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
Where is this file from? If not original? What's the license?

Reply:
That's from yarn source tree. We probably need some standard header for all Esy source files, right?

Reply:
We can add them at some point - if this is already from yarn, that's simple.